policyengine_us/tests/policy/baseline/gov/states/ut/tax/income/credits/retirement_credit/ut_retirement_credit_max.yaml

```diff
@@ -116,25 +116,3 @@
         state_code: UT
   output:  
     ut_retirement_credit_max: 662.5
-
-- name: Edge case test, birth year
```

Comment:
was this delete on purpose?

Reply:
I added it back in case it's useful. I am not sure I accept which version when there's a conflict in merging